charts/connect/README.md

```diff
@@ -112,6 +113,7 @@ helm install --set connect.applicationName=connect connect ./connect
 | operator.podAnnotations             | object     | `{}`                               | Additional annotations to be added to the Operator pods.                                                                                                                                           |
 | operator.podLabels                  | object     | `{}`                               | Additional labels to be added to the Operator pods.                                                                                                                                                |
 | operator.pollingInterval            | integer    | `600`                              | How often the 1Password Operator will poll for secrets updates.                                                                                                                                    |
+| operator.priorityClassName          | string | `""` | priorityClassName to apply to the Operator pods. |
```

Comment:
Suggestion to include a link to the relevant docs here so that folks unfamiliar with the concept can learn more if they so wish.

```suggestion
| operator.priorityClassName          | string | `""` | [priorityClassName](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/) to apply to the Operator pods. |
```